src/Tribe/Integrations/WPML/Views/V2/Filters.php

```diff
@@ -64,7 +64,8 @@ public static function translate_template_vars_urls( $template_vars = [] ) {
 		$url_keys = array_filter(
 			$template_vars,
 			static function ( $value, $key ) {
-				return str_contains( $key, 'url' ) && (bool) filter_var( $value, FILTER_VALIDATE_URL );
+				return strpos( $key, 'url' ) !== false
+				       && (bool) filter_var( $value, FILTER_VALIDATE_URL );
```

Comment:
* :warning: Found precision alignment of 3 spaces.
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: f9ff22d755d5b573279ca0ecdc9e9576</sub>